src/views/Correction/BenCorrection.vue

```diff
@@ -37,15 +37,17 @@
       validate="true"
     />
 
-    <CertifySection
-      class="mt-10"
-      :sectionNumber="isClientErrorCorrection ? '3.' : '2.'"
-      validate="true"
-    />
+    <template v-if="isClientErrorCorrection">
+      <CertifySection
+        class="mt-10"
+        :sectionNumber="isClientErrorCorrection ? '3.' : '2.'"
+        validate="true"
+      />
+    </template>
```

Comment:
Was this confirmed by Mihai?

Reply:
Yep. I confirmed with him during the standup today. We want it to be the same as the firm corrections.